### PR TITLE
Update to bevy 0.15

### PR DESCRIPTION
**Pull request by dmlukichev**
Updated library code, examples and documentation to work with bevy 0.15-rc.3

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
+.idea
```
Comment (Plonq) [inline .gitignore:1]: This should go in your [global gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/)

**Comment by Plonq** [comment]
Hey, thanks for this, looks like significant changes which saves me time 😄 

I won't merge until Bevy 15 is fully released, but this is a good head start.

**Review by Plonq — CHANGES_REQUESTED** [review]
Please fix the failed checks

**Comment by dmlukichev** [comment]
Formatting fixed, gitignore changes reverted.
I was hoping that you will publish rc version also, but it's up to you to decide.
Thank you for your lib!

**Comment by Plonq** [comment]
> I was hoping that you will publish rc version also, but it's up to you to decide.

I hadn't thought of that, but the egui feature is broken until `bevy_egui` is updated, so I don't think it's appropriate to release it as RC. Unless `bevy_egui` puts out an RC version It'll have to wait.

**Comment by Plonq** [comment]
Waiting for https://github.com/vladbat00/bevy_egui/pull/309

**Comment by pta2002** [comment]
@Plonq that PR is merged and `bevy_egui` 0.31 is now released with official support for bevy 0.15. Does that mean that this is ready to get merged?

**Comment by Plonq** [comment]
@dmlukichev Can you do the following please:

* Update deps (bevy to 0.15 and bevy_egui)
* Revert the version and changelog updates (I prefer to do them separately)

**Comment by dmlukichev** [comment]
Hi, @Plonq. Done.

**Comment by Plonq** [comment]
For some reason I am unable to push changes to this PR. You need to add `bevy_window` feature to the prod bevy dependency to fix the build.

**Comment by mineichen** [comment]
Instead of adding Camera3d() to every location, PanOrbitCamera is added, you could add Camera3D as a "required component" and remove `Camera3d::default()` from the examples.

In Plugin::build
```rust
app.register_required_components::<PanOrbitCamera, Camera3d>()
```

**Comment by dmlukichev** [comment]
Guys, sorry. Not in a position to argue.
It looks like like @mineichen tracks bevy's changes better that I do. I hope he will update this MR accordingly.
Best of luck, Dima

**Comment by Plonq** [comment]
@dmlukichev Don't worry about that, let's just get this PR fixed and merged.

@mineichen It's a good idea, feel free to open a new PR, or I will later

**Comment by Plonq** [comment]
I made the necessary changes in https://github.com/Plonq/bevy_panorbit_camera/pull/90, so I'll close this one.

**Pull request closed without merging**